commitlint.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
         // Subject
         'subject-empty': [2, 'never'],
         'subject-full-stop': [2, 'never', '.'],
-        'subject-max-length': [2, 'always', 100],
+        'subject-max-length': [2, 'always', 130],
```

Comment:
Why is it changed?

Reply:
I want it bigger, checked with my 13" screen, to see when github gonna move to the next line.

Reply:
Some time ago we agree to 120 symbols. Can we stick to this value?

Reply:
we can stick to it,

next time, if this keep wasting time for me, I'll gonna completely remove the commitlint 
it doesn't really contribute to people writing good commit messages...